src/NUnitFramework/tests/Constraints/PathConstraintTests.cs

```diff
@@ -28,7 +28,7 @@ namespace NUnit.Framework.Constraints
     /// <summary>
     /// Summary description for PathConstraintTests.
     /// </summary>]
-    [TestFixture]
+    [TestFixture, Platform("Win")]
```

Comment:
The tests in this class looked like they should be windows/linux specific anyway. Given the OS's have different rules around path case-sensitivity, making these tests OS specific seemed like the more sensible option.